### ISSUE TITLE
Latin Unicode to GSM-7 compatible python string as a standalone function

### ISSUE THREAD

**Issue by Wissotsky** [SkyDiverCool/py-smsify#5]
- [x] Have a simple conversion function accessible so people don't have to instantiate an object for a simple conversion.

- [ ] Additionally expose it as a python module that can be used with `python -m`

**Comment by Wissotsky** [comment]
Function accessible under `SmsMessage.encode(string: str) -> str` 

Module TBD

**Comment by Rashicom** [comment]
any one working on this ?

**Comment by Wissotsky** [comment]
> any one working on this ?

No, I don't use it currently. If you need to use it I recommend just forking it.
